modules/kubernetes_node_pools/variables.tf

```diff
@@ -70,4 +72,23 @@ variable "regional_cluster" {
   default     = false
   description = "Set to `true` to create node pool for regional cluster."
 }
+variable "no_execute_taint" {
+  type        = list(map(any))
+  description = "Object containing node NoExecute taints"
 
+  default = [{
```

Comment:
Defaults should be imho empty list, and only description could contain how to do it.

Reply:
actually those settings are quite static for NoSchedule and NoExecute. 99% of cases will  be default. The variable only gives the possibility to overwrite it.